lib/parse.js

```diff
@@ -32,7 +32,8 @@ CSSOM.parse = function parse(token) {
 		"importRule-begin": true,
 		"importRule": true,
 		"atBlock": true,
-		'documentRule-begin': true
+		'documentRule-begin': true,
+		"value-parenthesis": true
```

Comment:
this used in test ".gradient{background: -moz-linear-gradient(/*);*/top, #1E5799 0%, #7db9e8 100%)}" 
I think, this is correct for this test